crates/services/gas_price_service/src/fuel_gas_price_updater/tests.rs

```diff
@@ -42,64 +42,41 @@ impl FakeMetadata {
 
 #[async_trait::async_trait]
 impl MetadataStorage for FakeMetadata {
-    async fn get_metadata(&self) -> Result<Option<UpdaterMetadata>> {
+    async fn get_metadata(
+        &self,
+        _block_height: &BlockHeight,
+    ) -> Result<Option<UpdaterMetadata>> {
         Ok(self.inner.lock().await.clone())
     }
 
-    async fn set_metadata(&self, metadata: UpdaterMetadata) -> Result<()> {
+    async fn set_metadata(&mut self, metadata: UpdaterMetadata) -> Result<()> {
         let _ = self.inner.lock().await.replace(metadata);
```

Comment:
The same here, do we need to use lock, if you use `&mut self`?

Reply:
Same for here.